owncloudApp/src/main/res/layout/upload_list_item.xml

```diff
@@ -93,14 +95,44 @@
             tools:text="@string/auth_username"
             android:textSize="12dip" />
 
-        <TextView
-            android:id="@+id/upload_remote_path"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:textColor="@color/list_item_lastmod_and_filesize_text"
-            android:ellipsize="middle"
-            android:singleLine="true"
-            android:textSize="12dip" />
+            android:gravity="center_vertical">
+
+            <ImageView
+                android:id="@+id/space_icon"
+                android:layout_width="15dp"
+                android:layout_height="15dp"
+                android:layout_marginEnd="2dp"
+                android:src="@drawable/ic_spaces"
+                android:visibility="gone"
+                app:tint="@color/list_item_lastmod_and_filesize_text"/>
+
+            <TextView
+                android:id="@+id/upload_space_name"
+                android:layout_width="wrap_content"
+                android:maxWidth="100dp"
+                android:layout_height="wrap_content"
+                android:layout_marginEnd="@dimen/standard_half_margin"
+                android:textColor="@color/list_item_lastmod_and_filesize_text"
+                android:textStyle="bold"
+                android:ellipsize="middle"
+                android:singleLine="true"
+                android:textSize="12dip"
```

Comment:
I would consider using `sp` instead of `dip` since its the way to go with text sizes, same for other textSizes within this layout.

Reply:
It was legacy stuff so I decided to let it as it was, but checking that visually 12dip == 12sp, I changed it